src/CovidAppointmentTable.js

```diff
@@ -11,7 +11,7 @@ import HelpDialog from "./components/HelpDialog";
 import Loader from "react-loader";
 import MoreInformation from "./components/MoreInformation";
 import React, { useState, useEffect } from "react";
-import SignUpLink from "./components/SignUpLink";
+import SignUpLink, { HasSignUpLink } from "./components/SignUpLink";
```

Comment:
do you mind changing it to hasSignUpLink instead of HasSignUpLink? Generally I only do ThisKindOfCamelCase for components, and thisKind for functions and variables. Happy to fight it out though ;)